addons/product/models/product_template.py

```diff
@@ -291,15 +291,15 @@ def _get_weight_uom_id_from_ir_config_parameter(self):
     @api.model
     def _get_length_uom_id_from_ir_config_parameter(self):
         """ Get the unit of measure to interpret the `length`, 'width', 'height' field.
-        By default, we considerer that length are expressed in meters. Users can configure
+        By default, we considerer that length are expressed in millimeters. Users can configure
         to express them in feet by adding an ir.config_parameter record with "product.volume_in_cubic_feet"
         as key and "1" as value.
         """
         product_length_in_feet_param = self.env['ir.config_parameter'].sudo().get_param('product.volume_in_cubic_feet')
```

Comment:
I have still one issue with this task/PR, the Volume isn't now consistent with the length:
when `product_volume_volume_in_cubic_feet` == 'Cubic Meters', the volume is in m³ but the length is in mm, and when you change it in 'Cubic Feet' is the volume and the length become consistent.
What do you think about that ? 

(Your runbot is red, you should try to rebase all your branch)